lhotse/kaldi.py

```diff
@@ -47,7 +64,7 @@ def load_kaldi_data_dir(
                 )
             ],
             sampling_rate=sampling_rate,
-            num_samples=int(durations[recording_id] * sampling_rate),
+            num_samples=int(durations[recording_id]* sampling_rate),
```

Comment:
Nit: There should be a space before `*`.

Reply:
Done!